spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-focus-loss-tracker.spec.js

```diff
@@ -0,0 +1,233 @@
+/* eslint-env jasmine */
+
+describe('GA4 focus loss tracker', function () {
+  var GOVUK = window.GOVUK
+  var element
+  var expected
+
+  function agreeToCookies () {
```

Comment:
Non-blocking: I think we're using this kind of function in a few places now - worth raising an issue/adding a ticket to see if we can define it once in a helper somewhere instead of repeating it?

Reply:
Sounds good 👍